.github/workflows/release.yml

```
@@ -247,29 +226,16 @@ jobs:
         with:
           node-version: "20"
 
-      - name: Get latest emscripten LLVM release artifact
-        id: get-llvm-emscripten-artifact
-        uses: actions/github-script@v7
+      - name: get llvm gnu
```

```suggestion
      - name: get llvm gnu
```

I see emscripten below